.github/workflows/get_layer_info_for_nf_imgs.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Nextflow-approved image's layers
+on:
+  push:
+    branches: master
+    paths:
+      - 'nextflow-base-images/allowed_base_images.txt' #Runs every time this file is updated
+  schedule:
+    - cron: '0 0 * * 0'  # Run at midnight every Sunday
+
+jobs:
+  get_image_layers:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Pull HTTP Response
+        run: |
+          url_list=$(curl -s "https://raw.githubusercontent.com/uc-cdis/containers/master/nextflow-base-images/allowed_base_images.txt")
+          TOKEN=$(curl -s https://public.ecr.aws/token/ | jq -r .token)
+          layer_json="{}"
+          while IFS= read -r image_url; do
+              manifest_url=$(echo ${image_url} | sed 's|public\.ecr\.aws/\(.*\):\(.*\)|https://public.ecr.aws/v2/\1/manifests/\2|')
+              tag_name=$(echo ${image_url} | sed 's|\(.*\):\(.*\)|\2|')
+              response=$(curl -s -H "Authorization: Bearer $TOKEN" $manifest_url | jq "[.layers[].digest]")
```

Comment:
can we `echo` the manifest url and tag name before running the curl?

Reply:
Absolutely